src/MISC/fix_deposit.cpp

```diff
@@ -766,6 +776,13 @@ void FixDeposit::options(int narg, char **arg)
       vzlo = force->numeric(FLERR,arg[iarg+1]);
       vzhi = force->numeric(FLERR,arg[iarg+2]);
       iarg += 3;
+    } else if (strcmp(arg[iarg],"orientation") == 0) {
+      if (iarg+4 > narg) error->all(FLERR,"Illegal fix deposit command");
+      rflag = 1;
```

Comment:
There are many flags in this fix.  Hence it needs a better name than rflag.  E.g. rotationflag.